playground/pages/parse.html

```diff
@@ -0,0 +1,85 @@
+<!doctype html>
```

Comment:
For discoverability and sharing code do you think that it'd be reasonable to have only one landing page for the various tools and and sort of "tabs" between them to explore different functionality?

Reply:
Can do. Are you thinking of one HTML page, or multiple pages that link to each other? Having multiple makes it easier to bookmark a specific tool, though I guess I could use the `#fragment` and some JS to make that possible with a single page.

Reply:
I was thinking one page, but I'm not wed to the idea either. It mostly just seemed like there was some duplication between the styles/js/etc. Having links to each page from the other I think is also fine.

Reply:
I added links from each page to the other. There's not really much duplicated that isn't already shared - they use the same core stylesheet and worker, it's just the actual relevant UI which differs.